hedera-node/hedera-app/src/main/java/com/hedera/node/app/bbm/StateDumper.java

```diff
@@ -54,6 +60,7 @@
 public class StateDumper {
     private static final String SEMANTIC_UNIQUE_TOKENS = "uniqueTokens.txt";
     private static final String SEMANTIC_TOKEN_RELATIONS = "tokenRelations.txt";
+    private static final String SEMANTIC_ACCOUNTS = "accounts.txt";
```

Comment:
I don't know how I didn't see this comment yesterday. Anyways: since we've used this pattern on all dumpers maybe it would be better to create a new story that we can tackle at the end and update all the existing dumpers. What do you think?

Reply:
sure, do it as a separate issue please to cover all (final) dumpers.

Reply:
@david-bakin-sl, Jivko and I were discussing this internally and it seems that the files won't conflict because when we are creating the folder we are prefixing it with **mod** or **mono** and we are also adding the timestamp of the consensus of the last handled txn. E.g:
`final var dumpLoc = getExtantDumpLoc("mono", networkContext.consensusTimeOfLastHandledTxn());`

Do you think we still need to prefix the filenames?

Reply:
Afaik you're correct @vtronkov

Reply:
yes TY I didn't see that, so: NVM